ARImageTrackingExamples/ARImageTrackingExamples/View Controllers/SolidCardAnimationViewController.swift

```diff
@@ -0,0 +1,133 @@
+import Foundation
+import UIKit
+import ARKit
+
+class SolidCardAnimationViewController: UIViewController, ARExperimentSessionHandler {
+
+    @IBOutlet var sceneView: ARSCNView!
+    private let arSessionDelegate = ARExperimentSession()
+    private let arViewModel = ARViewModel()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        sceneView.delegate = self
+        arSessionDelegate.sessionHandler = self
+        sceneView.session.delegate = arSessionDelegate
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        let configuration = arViewModel.imageTrackingConfiguration()
+        sceneView.session.run(configuration, options: [.removeExistingAnchors, .resetTracking])
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        sceneView.session.pause()
+    }
+}
+
+extension SolidCardAnimationViewController: ARSCNViewDelegate {
+
+    func renderer(_ renderer: SCNSceneRenderer, didAdd node: SCNNode, for anchor: ARAnchor) {
+        guard let imageAnchor = anchor as? ARImageAnchor else {
+            return
+        }
+
+        guard let image = arViewModel.image(correspondingTo: imageAnchor.referenceImage) else {
+            print("could not find image")
+            return
+        }
+
+        let mainNode = SCNNode.mainNode(with: imageAnchor.imageSize(), colorBufferWriteMask: .alpha)
+
+        node.addChildNode(mainNode)
+        
+        let solidDataPlaneNode = SCNNode.cardNode(for: imageAnchor.imageSize(), withImage: image)
+        
+        mainNode.addChildNode(solidDataPlaneNode)
+        
+        if let webViewNode = SCNNode.webViewNode(for: imageAnchor.imageSize())
+        {
+            mainNode.addChildNode(webViewNode)
+        }
+    }
+}
+
+private extension SCNNode
+{
```

Comment:
pick a standard, on the same line everywhere, or next line everywhere. Don't mix ?